lang_GENERIC/analyze/naming_ast.ml

```diff
@@ -400,8 +400,9 @@ let resolve lang prog =
        | ImportAs (_, FileName (s, tok), Some alias) ->
           (* for Go *)
           let sid = Ast.gensym () in
-          let base = Filename.basename s, tok in
-          let resolved = ImportedModule (DottedName [base]), sid in
```

Comment:
The problem I was seeing is that Go does not let you write fully qualified module path.
You can't do import "html/template"
and then write as in Python html.template.foo(). You can just do template.foo(); That' the semantic of Go,
only the basename of the module is used to qualify.

Reply:
Based on [Go Import Decl spec](https://golang.org/ref/spec#Import_declarations), the package name is always a [fully qualified identifier ](https://golang.org/ref/spec#Qualified_identifiers). Thus, we may not need this str list as identifier.